automation_tools/__init__.py

```diff
@@ -1421,11 +1434,11 @@ def product_install(distribution, create_vm=False, certificate_url=None,
     if distribution in ('satellite6-downstream', 'satellite6-iso'):
         execute(java_workaround, host=host)
 
-    # Make sure downstream_install only is called for sat6.1 and sat6.2, as
-    # repofile_install and ak_install can be passed only for Sat6.3.
-    if satellite_version in ('6.1','6.2'):
-        if install_tasks[distribution] in ('repofile_install', 'ak_install'):
-            install_tasks[distribution] = 'downstream_install'
+    # Do not subscribe to cdn for Satellite6.3 if using Activation Keys of
```

Comment:
You comment is a bit confusing to me. 'Do not subscribe' means that the system is already subscribed, but you execute an `unsubscribe` task. I think what you want to say is:

```python
# If using the internal dog food server, unsubscribe from `CDN` first.
```